ui/apps/platform/src/Containers/Integrations/utils/tableColumnDescriptor.ts

```diff
@@ -89,6 +89,7 @@ const tableColumnDescriptor: Readonly<IntegrationTableColumnDescriptorMap> = {
         apitoken: [
             { accessor: 'name', Header: 'Name' },
             { accessor: 'role', Header: 'Role' },
+            { accessor: 'expiration', Header: 'Expiration' },
```

Comment:
It will be good to use consitent formatting as we do in other places:
https://github.com/stackrox/stackrox/blob/ce62d073a3cafce9f799f30b78cceff0d03d6a4a/ui/apps/platform/src/Containers/Violations/violationTableColumnDescriptors.tsx#L139

Reply:
I've checked some other places and a more common pattern is without '|'. For example,

* snoozed CVE: "<span>10/06/2023</span><span>1:11:11AM</span>"
* scan time for images: "<span>01/25/2024</span><span>2:37:29PM</span>"

So I think if we render the deadline (and not the days left), we should use the format with span and without '|'.